modules/setting/setting.go

```diff
@@ -348,7 +348,7 @@ var (
 	}{
 		EnableHardLineBreakInComments:  true,
 		EnableHardLineBreakInDocuments: false,
-		FileExtensions:                 strings.Split(".md,.markdown,.mdown,.mkd", ","),
+		FileExtensions:                 strings.Split(".md,.markdown,.mdown,.mkd,.livemd", ","),
```

Comment:
Please change it in `custom/conf/app.example.ini#1318` and `docs/content/doc/advanced/config-cheat-sheet.en-us.md` too.

Reply:
Huh? That whole paragraph is missing in the cheat sheet?
How did that happen?
Would you mind adding it now?

Reply:
I have added the new .extension to the docs and example